tests/.setup.js

```diff
@@ -3,6 +3,6 @@ import '@testing-library/jest-dom';
 
 window.resizeTo = (width, height) => {
     window.innerWidth = width || window.innerWidth;
-    window.innerHeight = width || window.innerHeight;
+    window.innerHeight = height || window.innerHeight;
```

Comment:
Oh, was this warned by the eslint? That's cool.

Reply:
yeah, I guess this was ignored for some reason before... Not sure why :-)